pkg/piperutils/FileUtils.go

```diff
@@ -3,11 +3,25 @@ package piperutils
 import (
 	"errors"
 	"io"
+	"io/ioutil"
 	"os"
 )
 
+// FileUtils ...
```

Comment:
What is the benefit of this comment?

Reply:
https://github.com/SAP/jenkins-library/pull/1176#discussion_r383127100